java/source/io.rst

```diff
@@ -33,30 +34,34 @@ Write - Out to File
     import org.apache.arrow.vector.VectorSchemaRoot;
     import static java.util.Arrays.asList;
     import org.apache.arrow.vector.ipc.ArrowFileWriter;
-
     import java.io.File;
     import java.io.FileOutputStream;
     import java.io.IOException;
 
-    try (RootAllocator rootAllocator = new RootAllocator(Long.MAX_VALUE)) {
+    try (
+        BufferAllocator allocator = new RootAllocator()
+    ) {
         Field name = new Field("name", FieldType.nullable(new ArrowType.Utf8()), null);
         Field age = new Field("age", FieldType.nullable(new ArrowType.Int(32, true)), null);
         Schema schemaPerson = new Schema(asList(name, age));
-        try(VectorSchemaRoot vectorSchemaRoot = VectorSchemaRoot.create(schemaPerson, rootAllocator)){
+        try(
+            VectorSchemaRoot vectorSchemaRoot = VectorSchemaRoot.create(schemaPerson, allocator);
             VarCharVector nameVector = (VarCharVector) vectorSchemaRoot.getVector("name");
+            IntVector ageVector = (IntVector) vectorSchemaRoot.getVector("age")
```

Comment:
You shouldn't need to close the vectors if they're part of a root, the root will close them